web-1.9/pom.xml

```diff
@@ -13,7 +13,7 @@
     <description>1.9 web project for coreapps</description>
 
     <properties>
-        <openMRSVersion>1.9.9</openMRSVersion>
+        <openMRSVersion>2.2.0</openMRSVersion>
```

Comment:
@ibacher  in order to intoduce (getPrimaryDiagnosis(encounter)), changed the core version of openmrs, though still it shows that it does not exist